R/pre-action-recipe.R

```diff
@@ -19,7 +19,9 @@
 #'
 #' @param x A workflow
 #'
-#' @param recipe A recipe created using [recipes::recipe()]
+#' @param recipe A recipe created using [recipes::recipe()]. The recipe
+#'    should have been trained already with [recipes::prep()]; workflows
+#'    will handle training internally.
```

Comment:
I think we need a "not" in here somewhere 😄

Reply:
ha, you're right!

Reply:
Addressed in [30b5fc7](https://github.com/tidymodels/workflows/pull/202/commits/30b5fc7b69e0e499b7c2bd6437a00dbf8a1f8f3d).🪱